system/modules/rest/models/RestService.php

```diff
@@ -1,33 +1,34 @@
 <?php
-class RestService extends DbService {
+class RestService extends SearchableService {
+
+	private $token;
 	
-	function getTokenJson($username, $password, $api) {
-		global $REST_API_KEY;
-		
-		if ($api && $api == $REST_API_KEY) {
-			$user = $this->w->Auth->login($username,$password,null,true);
-			if ($user) {
-				$session = new RestSession($this->w);
-				$session->setUser($user);
-				$session->insert();
-				return $this->successJson($session->token);
+	function checkModuleAccess($className) {
+		// STEVER: TODO I am having problems using Config. It is not returning the expected values as per config.php. Changes are not reflected. New values are ??????
```

Comment:
that was written a long time again. ignorance not cache.
S

On Wed, Feb 10, 2016 at 8:39 PM, Carsten Eckelmann <notifications@github.com
> wrote:

> In system/modules/rest/models/RestService.php
> <https://github.com/2pisoftware/cmfive/pull/180#discussion_r52434269>:
>
> >  	
> > -	function getTokenJson($username, $password, $api) {
> > -		global $REST_API_KEY;
> > -		
> > -		if ($api && $api == $REST_API_KEY) {
> > -			$user = $this->w->Auth->login($username,$password,null,true);
> > -			if ($user) {
> > -				$session = new RestSession($this->w);
> > -				$session->setUser($user);
> > -				$session->insert();
> > -				return $this->successJson($session->token);
> > +	function checkModuleAccess($className) {
> > +		// STEVER: TODO I am having problems using Config. It is not returning the expected values as per config.php. Changes are not reflected. New values are ??????
>
> have you cleared your config cache? :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/2pisoftware/cmfive/pull/180/files#r52434269>.
>



-- 
Steve Ryan
Software Engineer
stever@syntithenai.com
phone (mobile) 0423 083 685
phone (SIP)  61262237142
skype : irishflute
260 Spring Creek Road Buckajo NSW 2550
Australia